src/WebJobs.Extensions.DurableTask/DurableOrchestrationContextBase.cs

```diff
@@ -59,7 +59,11 @@ public abstract class DurableOrchestrationContextBase
         /// <value>
         /// <c>true</c> if the orchestrator function is currently being replayed; otherwise <c>false</c>.
         /// </value>
-        public virtual bool IsReplaying => false;
+        public virtual bool IsReplaying
+        {
+            get => false;
+            internal set => this.IsReplaying = value;
+        }
```

Comment:
This looks like a potential `StackOverflowException`. Can we just do this?
```csharp
public virtual bool IsReplaying { get; internal set; }
```
I don't remember why it was previously hard-coded to `false` but I can't imagine it was necessary.